packages/web-components/src/components/rux-push-button/rux-push-button.scss

```diff
@@ -1,112 +1,175 @@
 :host {
+  /* reference */
+  --border-color: currentColor;
+  --border-width: 1px;
+
+  /* display */
+  display: inline-flex;
+
+  /* layout */
+  flex-flow: column wrap;
+  vertical-align: middle;
+
+  /* input */
+  cursor: pointer;
+  user-select: none;
+}
+
+:host([hidden]) {
+  display: none;
+}
+
+:host([disabled]) {
+  /* input */
+  cursor: not-allowed;
+}
+
+input {
+    display: none;
+}
+
+.rux-push-button {
+    display: flex;
+
+    /* layout */
+    align-items: center;
+    justify-content: center;
+
+    /* type */
+    font: var(--button-type-shorthand);
+    letter-spacing: var(--font-body-1-letter-spacing);
+
+    /* input */
     cursor: pointer;
-    display: block;
-    margin: 0 2px;
-    -webkit-user-select: none;
-    -moz-user-select: none;
-    -ms-user-select: none;
     user-select: none;
 
-    &[disabled] {
-        opacity: var(--disabled-opacity);
-        cursor: not-allowed;
+    /* color */
+    background-color: var(--color-background-base-default);
```

Comment:
where did we land on the transparent bg colors? is this something we should bring up and align with design on?